Northstar.Custom/mod/scripts/vscripts/sh_server_dialog.gnut

```diff
@@ -0,0 +1,259 @@
+#if SERVER
+global function ServerDialogInit
+global function SendServerDialog
+global function SendMessageServerDialog
+global function AddDialogButton
+#elseif CLIENT
+global function ServerDialogueOnClientInit
+#elseif UI
+global function ClientCallBack_RequestDialog
```

Comment:
This should probably be called `ClientCallback_RequestDialog` (or dialogue if we are feeling particularly british ig) as Callback is one word